### ISSUE TITLE
Python Azure Function deployment fails with Remote builde failed! - source: filename argument required

### ISSUE THREAD

**Issue by claitonbecoco** [microsoft/Oryx#795]
### Bug Report

We have a Python Azure Function App we run on a Consumption Plan, and for the past 3 week the deployment from our local environment started to fail with the message 
`Remote build failed!`
And the inner message:
`/tmp/BuildScriptGenerator/161dc4ea83d94d2fb8340c3ec25a5e85/build.sh: line 91: source: filename argument required`

We tried to create a new Function App, both Consumption Plan or App Service Plan. Using Git deployment, VS Code Deployment or manual deployment from the terminal, all result on the same error it runs Oryx.

The last approach we took was to disable Oryx at all by setting `ENABLE_ORYX_BUILD` to `false` which does make the deployment succeed, but unfortunately the function then returns 500 (HTTP Trigger) with no message on logs apart from failure and the execution GUID.

This means that for the past 3 weeks we are unable to update our Azure function with the latest version, or even creating a replacement Function App.

The function runs completely fine on our local dev environment.

The way we been trying and repro the problem is doing manual deployment, this is the latest execution and full error message:

```func azure functionapp publish [FUNCTIONAPPNAME]
Getting site publishing info...
Creating archive for current directory...
Performing remote build for functions project.
Deleting the old .python_packages directory
Uploading 297.62 MB [#############################################################################]
Remote build in progress, please wait...
Updating submodules.
Preparing deployment for commit id '986b4caab4'.
Repository path is /tmp/zipdeploy/extracted
Running oryx build...
Command: oryx build /tmp/zipdeploy/extracted -o /home/site/wwwroot --platform python --platform-version 3.7 -p packagedir=.python_packages/lib/site-packages
Operation performed by Microsoft Oryx, https://github.com/Microsoft/Oryx
You can report issues at https://github.com/Microsoft/Oryx/issues

Oryx Version: 0.2.20200813.1, Commit: 16c4841161ae3c639fd26b0212be4f55e515ef7d, ReleaseTagName: 20200813.1

Build Operation ID: |hKWE+PHXRDU=.fd29b792_
Repository Commit : 986b4caab44e4bf7ad9e178bac9f217d

Detecting platforms...
Detected following platforms:
  python: 3.7.9


Source directory     : /tmp/zipdeploy/extracted
Destination directory: /home/site/wwwroot

/tmp/BuildScriptGenerator/161dc4ea83d94d2fb8340c3ec25a5e85/build.sh: line 91: source: filename argument required
source: usage: source filename [arguments]
/tmp/BuildScriptGenerator/161dc4ea83d94d2fb8340c3ec25a5e85/build.sh: line 91: source: filename argument required\nsource: usage: source filename [arguments]\n/opt/Kudu/Scripts/starter.sh oryx build /tmp/zipdeploy/extracted -o /home/site/wwwroot --platform python --platform-version 3.7 -p packagedir=.python_packages/lib/site-packages```


I would really appreciate some help on getting this one sorted, if there is a way to upgrade the Oryx version?

Kind regards,
Claiton Lovato

**Comment by kichalla** [comment]
@claitonbecoco Sorry to hear you are facing this issue. Could you please let me know if your repo has any Conda based `environment.yaml` file or `.ipynb` files? My suspicion is these files are being detected and Oryx is trying to create a conda envirnoment which is failing. Could you try removing these files and try again? We are currently working on a fix related to this.

**Comment by EdFarrell** [comment]
I do have *.ipynb files, but no environment.yaml files. The remote builds have been working with the .ipynb files for a while. However, I figured out the file that was breaking the build. It was a .ipynb file that had a file name with spaces in it. Once this file was added to the .funcignore file, the build passed again.

**Comment by claitonbecoco** [comment]
Thanks Kichalla and Ed.
We just did a major clean up on all our files, basically removing any other files that are not the pure Python code for the function and that seems to do the trick. it is strange as these files were there for a good time, but only started to fail now.
Happy to close this as we found a way around it. but if it works to test the new version you are working on I can leave it open too.
Kind regards,
Claiton
